src/services/shims.ts

```diff
@@ -322,8 +323,19 @@ module ts {
         }
     }
 
+    function serializeResultAsJson(target: Shim, key: string | symbol, descriptor: PropertyDescriptor) {

```

Comment:
it sure will be nice if you get an error applying a decorator that does not support symbol on a symbol-named property, but not on string-named properties.